driver/razerkbd_driver.c

```diff
@@ -3411,6 +3443,11 @@ static int razer_event(struct hid_device *hdev, struct hid_field *field, struct
     struct razer_kbd_device *device = hid_get_drvdata(hdev);
     const struct razer_key_translation *translation;
 
+    struct razer_kbd_usb_device_data *usbdev_data = razer_get_usb_device_data(device->usb_dev);
+    if(usbdev_data == NULL) {
+        return 1;
+    }
```

Comment:
Applies to most of these `usbdev_data == NULL` checks you added - is it expected that usbdev_data is NULL at some point, or should it never be null and you just added this to be 100% safe?

If it's so, either remove it (maybe not the best choice in case it can happen under extreme circumstances) or put a `WARN_ON` in there so people see it in the dmesg if this happens and it's not just silently failing